apps/src/code-studio/header.js

```diff
@@ -172,6 +173,8 @@ function shareProject() {
     const appType = dashboard.project.getStandaloneApp();
     const pageConstants = getStore().getState().pageConstants;
     const canShareSocial = !pageConstants.isSignedIn || pageConstants.is13Plus;
+    const canPublish = !!appOptions.isSignedIn &&
+      PUBLISHED_PROJECT_TYPES.includes(appType);
```

Comment:
The `!!` boolean coercion on `appOptions.isSignedIn` is redundant since it's already inside a `&&` boolean expression.